include/nccl_ofi_tracepoint.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022 Amazon.com, Inc. or its affiliates. All rights reserved.
+ */
+
+
+#pragma once
```

Comment:
In the rest of the code we use include guards, probably because pragma once is not standard. 
Regardless of which one is better, let's be consistent.